### PR TITLE
Minor Readme updates

### PR DESCRIPTION
**Pull request by lemunozm**
Changes:
- Added prerequisites to the Run section.

To review:
- Section "Generating a new genesis file", no longer has `id`/`protocol_id` to change in `chain_spec.rs`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +49,7 @@ Start  centrifuge-chain as parachain
 ./scripts/init.sh start-parachain
 ```
 
-Note: above command will show logs and block until parachain is stopped  
+Note: above command will show logs and block until parachain is stopped
```
Comment (NunoAlexandre) [inline README.md:52]:
```suggestion
Note: the command above will show logs and block until the parachain is stopped
```

File: README.md [hunk 2]
```diff
@@ -129,7 +133,7 @@ the collator node the parachain will be running on.
 
 # Upgrading to a newer Polkadot / Substrate / Cumulus version
 
-When a new version of Polkadot is released, a mirroring release hapens for the other 
+When a new version of Polkadot is released, a mirroring release hapens for the other
```
Comment (NunoAlexandre) [inline README.md:136]: Found a typo on my original text 🙈 

```suggestion
When a new version of Polkadot is released, a mirroring release happens for the other
```

**Review by NunoAlexandre — COMMENTED** [review]
Sweet! Just two small things and we get it in

File: README.md (hunk 2 above)
Comment (lemunozm) [reply README.md:136]: It usually hapens 😛

**Review by NunoAlexandre — APPROVED** [review]
Looks good, thanks @lemunozm 💯 

Don't forget to remove the "WIP - " prefix before merging 👍

**Comment by NunoAlexandre** [comment]
>To review:
>
> > Section "Generating a new genesis file", no longer has id/protocol_id to change in chain_spec.rs


Good catch, @lemunozm. I have never used that section/command so I'd say that we can actually just drop it. Let's hear from @mikiquantum on that tho

**Comment by mikiquantum** [comment]
@lemunozm thanks for going over this, feel free to remove that section.

**Review by mikiquantum — APPROVED** [review]
👍

**Review by mikiquantum — APPROVED** [review]
👍

**Pull request merged**